[2024-01-30T17:37:13.298+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T17:00:00+00:00 [queued]>
[2024-01-30T17:37:13.400+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T17:00:00+00:00 [queued]>
[2024-01-30T17:37:13.405+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T17:37:13.568+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-27 17:00:00+00:00
[2024-01-30T17:37:13.575+0000] {standard_task_runner.py:60} INFO - Started process 2602 to run task
[2024-01-30T17:37:13.628+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-27T17:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmp2p21pdxv']
[2024-01-30T17:37:13.643+0000] {standard_task_runner.py:88} INFO - Job 24: Subtask get_weather_data
[2024-01-30T17:37:14.013+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T17:00:00+00:00 [running]> on host 68c8327b56d3
[2024-01-30T17:37:14.961+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-27T17:00:00+00:00'
[2024-01-30T17:37:16.214+0000] {logging_mixin.py:188} INFO - {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}
[2024-01-30T17:37:19.467+0000] {logging_mixin.py:188} INFO - {'record': {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}, 'metadata': {'id': '65b933cf1f5677401f28736a', 'createdAt': '2024-01-30T17:37:19.460Z', 'private': True, 'name': 'weather_77569860-7f3d-476c-a2c8-540ee1476cc9', 'collectionId': '65ab4466266cfc3fde7c6f31'}}
[2024-01-30T17:37:19.507+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-30T17:37:19.583+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240127T170000, start_date=20240130T173713, end_date=20240130T173719
[2024-01-30T17:37:19.652+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T17:37:19.731+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
